test/vspec/documentation.vim

```diff
@@ -15,7 +15,16 @@ describe 'documentation docstrings'
         normal GK
         Expect bufname('%') == "__doc__"
         Expect &filetype == 'rst'
-        let content = join(getline(1,'$'), "\n")
+        let header = getline(1, 2)
+        PythonJedi vim.vars["is_py2"] = sys.version_info[0] == 2
+        if g:is_py2
+            Expect header[0] == "Docstring for __builtin__:class ImportError"
+            Expect header[1] == "==========================================="
+        else
+            Expect header[0] == "Docstring for builtins:class ImportError"
+            Expect header[1] == "========================================"
+        endif
+        let content = join(getline(3, '$'), "\n")
         Expect stridx(content, "Import can't find module") > 0
```

Comment:
@davidhalter with Jedi master "content" is "ImportError(*args, name: Optional[str]=..., path: Optional[str]=...)" here, i.e. the signature itself?!
git-bisected to https://github.com/davidhalter/jedi/commit/4161bfc7f28454db2a307cdcae463e41bec3103d

Reply:
@blueyed For whatever reason, the documentation of some builtins doesn't seem to work anymore and the tests don't catch it. No idea why, I will investigate and fix it. I have noticed it a few days ago when documentation for open didn't show up. So really a Jedi bug.

Reply:
It works with the submodule's version, doesn't it?

Reply:
I.e. the test fails locally when updating Jedi to the above commit.
Going to merge this for now.

Reply:
Make sense, thanks!